opentsdb/tsdb.go

```diff
@@ -2,6 +2,7 @@
 package opentsdb // import "bosun.org/opentsdb"
 
 import (
+	"bosun.org/cache"
```

Comment:
Both this and the graphite packages should instead have their cache managed in their query functions in funcs.go (timeTSDBRequest, timeGraphiteRequest). That way the low levels packages don't have to know about the cache at all.